### ISSUE TITLE
Update readme

### ISSUE THREAD

**Issue by mayeaux** [tinderjs/tinder-desktop#123]
Explain bower is a dep for `npm install`
Explain using electron-prebuilt

Add new developers
Change the link directly to the heroku app for getting a slack invite

**Comment by SaturnusDJ** [comment]
Yeah if bower is not automatically installed by `npm install` it should be explained. Then again, @hanford was working on removing bower so might not be needed to explain bower.

**Comment by mayeaux** [comment]
Takes only 30 seconds to add it to the readme. I'll get this done sometime soon

**Comment by amosshapira** [comment]
Yep, I'm not familiar with the `npm` eco-system and had to figure out to manually execute `npm install bower` from the error message I got from `npm install` while I was following the instructions in the README.

I don't see any downside for adding this step until Bower dependency is removed.

**Comment by gdborton** [comment]
I believe that this can be closed. I see that bower is in the README.
